internal/libyaml/yaml.go

```diff
@@ -575,11 +575,9 @@ type AliasData struct {
 // Parser structure holds all information about the current
 // state of the parser.
 type Parser struct {
-	// Err handling
-	Err error
+	lastError error
```

Comment:
See https://github.com/yaml/go-yaml/pull/152/changes#r2643754224

I find strange to store an error here

Especially if we consider it's just for storing a state